common/changes/@cityofzion/wallet-connect-sdk-core/CU-8678g1zgg_2023-08-31-13-48.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cityofzion/wallet-connect-sdk-core",
+      "comment": "The methods encrypt, decrypt and decryptFromArray methods was implemented",
```

Corrigindo o inglês seria:
```suggestion
      "comment": "The encrypt, decrypt and decryptFromArray methods have been implemented",
```

Mas gostaria de acrescentar esse "New encryption methods" na frente
```suggestion
      "comment": "New encryption methods: The encrypt, decrypt, and encrypt From Array methods have been implemented",
```